agent/php_stack.c

```diff
@@ -455,7 +350,7 @@ void nr_php_backtrace_fd(int fd, int limit TSRMLS_DC) {
   ex = EG(current_execute_data);
 
   while (ex) {
-#ifdef PHP7
+#if ZEND_MODULE_API_NO >= ZEND_7_2_X_API_NO /* PHP 7.2+ */
```

Comment:
This should just be removed altogether, no?

Reply:
Good catch, fixed: [fdf400e](https://github.com/newrelic/newrelic-php-agent/pull/1081/commits/fdf400ed88340eeda007431816d23b19efd91c20)